### PR TITLE
Unified Colors

### PR DESCRIPTION
**Pull request by codethejason**
Organisers may now choose a unified color theme for their event application in the config.scss file.

### CODE REVIEW

File: assets/scss/settings/colors.scss [hunk 1]
```diff
@@ -26,7 +31,14 @@ $txt-clr: (
...
+    brown: $dark,
+    tan: $dark,
+    thistle: $dark,
+    banana: $dark,
 	dark: $light,
 	light: $dark
-);
\ No newline at end of file
+);
+
```
Comment (creativepsyco) [inline assets/scss/settings/colors.scss]: Dont commit sass cache files.

**Comment by SaptakS** [comment]
I think there is no need to commit the main.css as well as it will be generated automatically on running gulp.

**Comment by SaptakS** [comment]
@creativepsyco I guess the rest is good to go?

File: config.scss [hunk 2]
```diff
@@ -1,3 +1,26 @@
 $config: (
-	theme: default
-);
\ No newline at end of file
+	theme: default,
+);
+
```
Comment (codethejason) [inline config.scss]: There are no more // comments.

File: assets/scss/main.scss [hunk 3]
```diff
@@ -25,8 +25,8 @@
 		width: map-get($conf, size);
 		$border: map-get($conf, border);
 		$theme: map-get($conf, theme);
-		color: map-get($bg-clr, $theme);
-		border-color: map-get($bg-clr, $border);
+		color: map-get($txt-clr, $theme) !important;
```
Comment (codethejason) [inline assets/scss/main.scss]: Must be used in this case if I cannot modify JS files.

File: assets/scss/settings/colors.scss (hunk 1 above)
Comment (codethejason) [inline assets/scss/settings/colors.scss]: No more sass cache and no more /**/ comments. File now has trailing newline.

File: assets/scss/main.scss [hunk 4]
```diff
@@ -25,8 +25,8 @@
...
+		border-color: map-get($txt-clr, $border) !important;
```
Comment (codethejason) [inline assets/scss/main.scss]: Must be used in this case since I am only modifying sass files.

File: assets/scss/main.scss [hunk 5]
```diff
@@ -12,7 +12,7 @@
 	.#{map-get($conf, name)} {
 		width: map-get($conf, size);
 		$theme: map-get($conf, theme);
-		background: map-get($bg-clr, $theme);
+		background: map-get($bg-clr, $theme) !important;
```
Comment (codethejason) [inline assets/scss/main.scss]: Must be used in this case.

**Comment by championswimmer** [comment]
If this is all squashed into one commit would be good.

**Comment by SaptakS** [comment]
@championswimmer @creativepsyco I think this PR is good to be merged?

**Pull request merged**